client/src/dataset/components/HDF5ParamsForm.tsx

```diff
@@ -90,13 +98,27 @@ const HDF5ParamsForm: React.FC<MergedProps> = ({
                 {dsPathInput}
             </Form.Field>
             {warning}
+            <Reshape navShape={values.nav_shape} sigShape={values.sig_shape} syncOffset={values.sync_offset} hideInfo setFieldValue={setFieldValue} setFieldTouched={setFieldTouched} />
             <Button primary type="submit" disabled={isSubmitting}>Load Dataset</Button>
             <Button onClick={onCancel} >Cancel</Button>
             <Button type="button" onClick={handleReset}>Reset</Button>
         </Form>
     )
 }
 
+const getInfoItemForDSPath = (ds_path: string, info?: DatasetInfoHDF5) => {
```

Comment:
Would be good to have a return type annotation here:

```suggestion
const getInfoItemForDSPath = (ds_path: string, info?: DatasetInfoHDF5): DatasetInfoHDF5Item => {
```

Reply:
Thanks, I've added that in.